flink-runtime/src/main/java/org/apache/flink/runtime/memory/MemoryManager.java

```diff
@@ -89,6 +89,18 @@
 	/** Flag whether the close() has already been invoked. */
 	private volatile boolean isShutDown;
 
+	/**
+	 * Creates a memory manager with the given memory types, capacity and given page size.
+	 *
+	 * @param memorySizeByType The total size of the memory to be managed by this memory manager for each type (heap / off-heap).
+	 * @param pageSize The size of the pages handed out by the memory manager.
+	 */
+	public MemoryManager(
```

Comment:
is github messing up the commit order again? to me it shows that the numberOfSlots argument is removed in a prior commit.

Reply:
I will push again, hopefully it will fix the commit order, otherwise I will add commit order into the PR description